### ISSUE TITLE
Validation errors 

### ISSUE THREAD

**Issue by adriannelee** [cds-snc/notification-planning#497]
Issue group: Semantics
Visibility: 3
Evaluation: Does not support
Success Criteria: 4.1.1: Parsing
Needs design: 

### Description
Each page and state should be run through a [validator](https://validator.w3.org/nu/) to ensure the markup structure is correct. While most/all pages have validation issues, they tend to be related to invalid markup nesting (e.g a `<div>` in a `<span>`) or invalid attributes (e.g. “row” attribute on an `<input>`).

**Comment by andrewleith** [comment]
This relates to the work being done with cypress in #1212

**Comment by YedidaZalik** [comment]
Here is the [public website audit done by Élise](https://docs.google.com/spreadsheets/d/146lc_wAoI0OzO9YhTVFmfaK6NGHzfuNrVX0SzwldYkM/edit#gid=1280586311)
I'm still looking for the one that I did

**Comment by andrewleith** [comment]
Thanks @YedidaZalik, I was able to get several pages I missed from that list! 🎉

**Comment by andrewleith** [comment]
- Fixes have been split into 4 PRs, 2 of 4 have been approved.

**Comment by yaelberger-commits** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Notify-Planning-614b3ad91bc2030015ed22f5/issues/cds-snc/notification-planning/497?planning-poker) with Zenhub @adriannelee @amazingphilippe @andrewleith
